src/OrbitVersion/include/OrbitVersion/OrbitVersion.h

```diff
@@ -9,6 +9,53 @@
 
 namespace orbit_version {
 
+struct Version final {
+  Version() = default;
+  Version(const Version&) = default;
+  Version(Version&&) = default;
+  Version& operator=(const Version&) = default;
+  Version& operator=(Version&&) = default;
```

Comment:
I believe that's superfluous or am I missing a use-case?

Reply:
I never am sure if they are generated by default. Or do you mean we do not need them?

Reply:
I mean they are generated automatically.

Reply:
removed